peps/pep-0777.rst

```diff
@@ -0,0 +1,81 @@
+PEP: 777
+Title: Reinventing the Wheel (Wheel 2.0)
+Author: Ethan Smith <ethan@ethanhs.me>
+Status: Draft
+Type: Standards Track
+Topic: Packaging
+Created: 20-May-2024
+
+- Should mention https://discuss.python.org/t/speculative-wheel-2-0-and-migration-strategies/21600/13
```

Comment:
Let's start with a short text saying something like this is a placeholder and the PEP will be added in future.

Reply:
I'll write that up and push shortly, thank you! Wasn't sure how to handle that.